NetCoreDemos/Views/Toolbar/Multiline.cshtml

```diff
@@ -0,0 +1,315 @@
+﻿@model DevExtreme.NETCore.Demos.ViewModels.DropDownButtonViewModel
+
+@{
+    var toolbarModes = new [] { "Multiline mode", "Singleline mode" };
+    var listTypes = new [] {
+        new { icon = "orderedlist", alignment = "orderedlist", hint = "Ordered" },
+        new { icon = "bulletlist", alignment = "bulletlist", hint = "Bullet" }
+    };
+}
+
+<div>
+    @(Html.DevExtreme().Sortable()
+        .MinWidth(500)
+        .MinHeight(150)
+        .MaxHeight(370)
+        .Handles("right")
+        .Content(
+            Html.DevExtreme().Toolbar()
+                .ID("toolbar")
+                .Multiline(false)
+                .Items(items => {
+                    items
+                        .Add()
+                        .Widget(w => w
+                            .Button()
+                            .Icon("undo")
+                            .OnClick("undoButtonClick")
+                        )
+                        .Location(ToolbarItemLocation.Before)
+                        .LocateInMenu(ToolbarItemLocateInMenuMode.Never);
+                    
+                    items
+                        .Add()
+                        .Widget(w => w
+                            .Button()
+                            .Icon("redo")
+                            .OnClick("redoButtonClick")
+                        )
+                        .Location(ToolbarItemLocation.Before)
+                        .LocateInMenu(ToolbarItemLocateInMenuMode.Never);
+                    
+                    items
+                        .Add()
+                        .Template(@<text><div class="separator"></div></text>)
```

Comment:
let's move template to js function

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/90658d7e4e3add4f9fea2e1148c0a05badc7ee41

https://github.com/DevExpress/devextreme-demos/pull/2004/commits/7deb0a901223529b76ea02a256f42a91a74b9f64